packages/blocks/src/list-block/list-model.ts

```diff
@@ -10,6 +10,7 @@ export const ListBlockModelSchema = defineBlockSchema(
   () => ({
     type: 'bulleted' as ListType,
     checked: false,
+    open: null, // TODO consider more explicit prop naming like 'hideChildren'
```

Comment:
Wondering can we use `boolean` that defaults to `true` here?

Reply:
worth considering. i chose null because a new block will not have children, and thus cannot be "open"

Reply:
As i mentioned in the Todo on this line, i will consider changing the direction of the flag by renaming to hideChildren, and then it can be false by default.  

This will also be more compatible with the idea to make all list block flavors have hideable children.